pages/employeeSearch.js

```diff
@@ -58,7 +58,23 @@ export const getServerSideProps = async () => {
   // // const res = await fetch(`https://randomuser.me/api/`)
   // const Employees = await res.json()
 
-  const employees = [
+  let temp = await fetch("http://localhost:8080/therapist", {

```

Comment:
this would apply to all fetch requests, but may want to create a baseUrl variable to then fetch(baseUrl + "/therapist"), could place the var in the util file I mentioned above